crates/frontend/src/api.rs

```diff
@@ -45,14 +45,21 @@ pub async fn fetch_dimensions(
 
 // #[server(GetDefaultConfig, "/fxn", "GetJson")]
 pub async fn fetch_default_config(
-    filters: &PaginationParams,
+    pagination: &PaginationParams,
+    filters: &DefaultConfigFilters,
     tenant: String,
     org_id: String,
 ) -> Result<PaginatedResponse<DefaultConfig>, ServerFnError> {
     let client = reqwest::Client::new();
     let host = use_host_server();
 
-    let url = format!("{}/default-config?{}", host, filters.to_string());
+    let filters = filters.to_string();
+    let url = if filters.is_empty() {
+        format!("{}/default-config?{}", host, pagination)
+    } else {
+        format!("{}/default-config?{}&{}", host, pagination, filters)
+    };
```

Comment:
this will simply work
```suggestion
    let url = format!("{host}/default-config?{pagination}&{filters}");
```